docs/markdown/Python/python/pex.md

```diff
@@ -20,6 +20,61 @@ A Pex can be bundled into a single `.pex` file. This file, when executed, knows
 
 Check out [blog.pantsbuild.org/pants-pex-and-docker](https://blog.pantsbuild.org/pants-pex-and-docker/) for how this workflow gets even better when combined with Pants's Docker support!
 
+Building a Pex
+--------------
+
+You define a Pex using the [`pex_binary`](doc:reference-pex_binary) target type:
+
+```python path/to/mybinary/BUILD
+python_sources(name="lib")
+
+pex_binary(
+    name="bin",
+    dependencies=[":lib"],
+    execution_mode="venv",
+)
+```
+
+You then use the `package` goal to build the Pex, which will be output under the [dist/] directory.
+
+```shell
+$ pants package path/to/mybinary:bin
+```
+
+There are several fields you can set on a `pex_binary` to configure the layout, entry point and behavior of the resulting Pex.  See the [documentation](doc:reference-pex_binary) for details.
+
+
+Setting the target platforms for a Pex
+--------------------------------------
+
+By default, the `package` goal builds a Pex that runs on the local machine's architecture and OS, and on a locally found interpreter compatible with your code's [interpreter constraints](doc:python-interpreter-compatibility). However, you can also build a multiplatform Pex - one that will run on multiple architecture+OS+interpreter combinations.
+
+To do so, you must configure the [`complete_platforms`](doc:reference-pex_binary#codecomplete_platformscode) field on your `pex_binary` to point to `file` targets that provide detailed information about your target platforms. This is information that Pants can't determine itself because it's not running on those platforms:
+
+```python BUILD
+file(
+    name="platform1",
+    source="platform1.json",
+)
+
+file(
+    name="platform2",
+    source="platform2.json",
+)
```

Comment:
Did you consider `files(name="platforms", sources=["platform1.json", "platform2.json"])` with `complete_platforms=[":platforms"]`? I'm not sure sure whether multiple `file`s vs. all-in-one `files` is clearer for a reader!

Reply:
It seems to me `files` saves typing and is a pantsism better discussed in the BUILD docs, whereas `file` illustrates the key idea. You create platforms once per each one you target (the names could be better / more real-life to make this clearer), then you pick one or more per given binary depending if you need to target 1 platform or two or more. You might use files vs file to code golf or DRY or what have you, but that is not really relevant here it seems to me.

Reply:
I wanted to emphasize that complete_platforms is a list. I will pick more real-life names though, yeah.

Reply:
Done